services/ui-src/src/page/section/DetailSection.tsx

```diff
@@ -116,6 +125,34 @@ export const DetailSection = ({
                 </Review>
               )
           )}
+          {detail.reviewTeam && (
+            <Review key="srt" heading="Review Team (SRT)">
+              {detail.reviewTeam.length === 0
+                ? "-- --"
+                : showReviewTeam
+                ? detail.reviewTeam.map((reviewer) => (
+                    <>
+                      {reviewer}
+                      <br />
+                    </>
+                  ))
+                : detail.reviewTeam.slice(0, 3).map((reviewer) => (
+                    <>
+                      {reviewer}
+                      <br />
+                    </>
+                  ))}
+              {detail.reviewTeam.length > NUM_REVIEWERS_TO_SHOW && (
+                <Button
+                  className="review-team"
+                  aria-expanded="false"
+                  onClick={toggleReviewTeam}
+                >
+                  {showReviewTeam ? "View Less Names" : "View All Names"}
+                </Button>
+              )}
+            </Review>
```

Comment:
I like the fore thought about only showing the button if the size is greater...makes perfect sense and might be something I wouldnt have thought about until I ran into the condition live in the app!

Reply:
.... which is what happened :)  I was like "what's this?!" during my own testing